client/src/app/ui/modules/editor/components/editor/editor.component.ts

```diff
@@ -36,6 +37,48 @@ export class EditorComponent extends BaseFormControlComponent<string> implements
         relative_urls: false,
         link_title: false,
         height: 320,
+        style_formats: [
+            {
+                title: _(`Headers`),
+                items: [
+                    { title: _(`Header 1`), format: `h1` },
+                    { title: _(`Header 2`), format: `h2` },
+                    { title: _(`Header 3`), format: `h3` },
+                    { title: _(`Header 4`), format: `h4` },
+                    { title: _(`Header 5`), format: `h5` },
+                    { title: _(`Header 6`), format: `h6` }
+                ]
+            },
+            {
+                title: _(`Inline`),
+                items: [
+                    { title: _(`Bold`), icon: `bold`, format: `bold` },
+                    { title: _(`Italic`), icon: `italic`, format: `italic` },
+                    { title: _(`Underline`), icon: `underline`, format: `underline` },
+                    { title: _(`Strikethrough`), format: `strikethrough` },
+                    { title: _(`Superscript`), icon: `superscript`, format: `superscript` },
+                    { title: _(`Subscript`), icon: `subscript`, format: `subscript` },
+                    { title: _(`Code`), format: `code` }
+                ]
+            },
+            {
+                title: _(`Blocks`),
+                items: [
+                    { title: _(`Paragraph`), format: `p` },
+                    { title: _(`Div`), format: `div` },
+                    { title: _(`Pre`), format: `pre` }
+                ]
+            },
+            {
+                title: _(`Alignment`),
+                items: [
+                    { title: _(`Left`), format: `alignleft` },
+                    { title: _(`Center`), format: `aligncenter` },
+                    { title: _(`Right`), format: `alignright` },
+                    { title: _(`Justify`), format: `alignjustify` }
```

Comment:
Haven't looked too deeply into how this works with tinyMCE, but shouldn't the `alignjustify` item be removed if you want to remove the option?